### PR TITLE
ZMS-436 ephemeral storage information

### PR DESCRIPTION
**Pull request by tmclane**
This is a start at the Ephemeral Data documentation.  Would like input on what I have and I will make any requested changes before actually merging.

### CODE REVIEW

**Comment by fatkudu** [comment]
I suspect we should be pushing to a dev branch or even a zms-dev branch.  Perhaps the current master should be tagged in some way to show that it is the docs for 8.7

File: glossary.adoc [hunk 1]
```diff
@@ -84,6 +84,9 @@ Edge MTA::
 Entry::
     An item in the directory server, such as an account or mail host.
 
+Ephemeral Data::
+    Data which is of ephemeral in nature. Login timestamps, authentication tokens, etc.
```
Comment (fatkudu) [inline glossary.adoc]: Perhaps `Data which is short lived or fast changing in nature`

**Comment by tmclane** [comment]
I agree on the pushing to 'zms-dev' branch. I can't create it however and I had forgotten to ask John E for permissions to be able to do so.

File: glossary.adoc (hunk 1 above)
Comment (tmclane) [inline glossary.adoc]: Done.

**Comment by fatkudu** [comment]
+1 - Looks good to me.

**Pull request closed without merging**